src/ngraph/runtime/cpu/op/lstm.hpp

```diff
@@ -32,41 +32,29 @@ namespace ngraph
             CPU_BACKEND_API
             static constexpr NodeTypeInfo type_info{"Lstm", 0};
             const NodeTypeInfo& get_type_info() const override { return type_info; }
-// INPUTS:
-// [0] - {Xt} input tensor of layout TNC, Shape{sequence length*batch_size,
-//       feature_size}
-// [1] - recurrent state tensors {ht_1 | ct_1} of Shape{sequence length*batch_size,
-//       feature_size}
-// [2] - initializer for the input weights matrix, used for the linear transformation of
-//       the inputs.
-// [3] - initializer for the recurrent weights matrix, used for the linear
-//       transformation of the recurrent state.
-// [4] - Initializer for the bias vector w.r.to inputs + hidden state (ibh_bias +
-//       hbh_bias)
+            // INPUTS:
+            // [0] - {Xt} input tensor of layout TNC, Shape{sequence length*batch_size,
+            //       feature_size}
+            // [1] - recurrent state tensors {ht_1 | ct_1} of Shape{sequence length*batch_size,
+            //       feature_size}
+            // [2] - initializer for the input weights matrix, used for the linear transformation of
+            //       the inputs.
+            // [3] - initializer for the recurrent weights matrix, used for the linear
+            //       transformation of the recurrent state.
+            // [4] - Initializer for the bias vector w.r.to inputs + hidden state (ibh_bias +
+            //       hbh_bias)
```

Comment:
not in scope of this review but we will need to check with Pruthvi about the documentation here. the new Lstm op has more args